delta_utils/geocoding.py

```diff
@@ -22,6 +22,21 @@ def lookup_country(
     fields: Optional[List[str]] = None,
     country_fields: Optional[List[str]] = None,
 ):
+    """
+    Returns a dictionary with all settings from the path
```

Comment:
In multiple places, you have a "Returns ..." sentence and a "Returns: ..." as well, can you remove these "Returns" so we don't have multiple definitions of what the function returns?